<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteEmailDialog</class>
 <widget class="QDialog" name="WriteEmailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="head" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#head{background-color: rgb(66,74,89);}
#send,#save,#accessory{border:none;background-color: rgb(66,74,89);color:white;}
#send:hover,#accessory:hover,#save:hover{background-color: rgb(90,90,90);}

</string>
   </property>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/发送.png</normaloff>:/souce/souce/发送.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="accessory">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>附件</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/附件.png</normaloff>:/souce/souce/附件.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>存入草稿</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/保存.png</normaloff>:/souce/souce/保存.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="writeClose">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#writeClose{
	background-color:tomato;
	border-radius:10px;}
#writeClose:hover{border-image: url(:/souce/souce/关闭1.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="writeMin">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">#writeMin{
background-color:rgb(255,224,81);
	border-radius:10px;}
#writeMin:hover{
border-image: url(:/souce/souce/最小化.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>39</y>
     <width>720</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="subject">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>主 题:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subjectEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="receiver">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>收件人:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="receiverEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="attachShowBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>2</y>
      <width>331</width>
      <height>87</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#attachShowBox{
border:none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:tomato;
font: 75 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>已添加附件</string>
     </property>
    </widget>
    <widget class="QListWidget" name="attachList">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>0</y>
       <width>251</width>
       <height>87</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#attachList{
border:none;
}
#attachList::Item{
height:30px;
}
#attachList::Item:hover{
background-color: rgb(41, 189, 139);
}</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="emailContent">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>199</y>
     <width>720</width>
     <height>361</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QWebView" name="richEmailEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>721</width>
     <height>411</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>file:///C:/Users/14356/Desktop/EmailSystem_搜索_转发_回复_12.15/Email/Libs/kindeditor-4.1.7/examples/default.html</string>
    </url>
   </property>
  </widget>
  <widget class="QLabel" name="loading">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>receiverEdit</tabstop>
  <tabstop>subjectEdit</tabstop>
  <tabstop>send</tabstop>
  <tabstop>accessory</tabstop>
  <tabstop>save</tabstop>
  <tabstop>richEmailEdit</tabstop>
  <tabstop>emailContent</tabstop>
 </tabstops>
 <resources>
  <include location="souce.qrc"/>
 </resources>
 <connections/>
</ui>
